lib/req-res/status.js

```diff
@@ -8,23 +8,16 @@ module.exports = function ( options ) {
 	return {
 		'success' : function ( data ) {
 
-			log.info( 'success', data );
-
 			respond( {
 				'status' : 'success',
 				'data'   : data
 			} );
+
+			log.info( 'success', data );
```

Comment:
Should we add the `options.messageType` and possibly `options.message` here too? It would be nice to see what messageType the fail came from.